docs/usage/control-plane-endpoints-and-ports.md

```diff
@@ -0,0 +1,46 @@
+# Endpoints and Ports of a Shoot Control-Plane
+
+With the reversed VPN tunnel, there are no endpoints with open ports in the shoot cluster required by gardener.
+In order to allow communication to the shoots control-plane in the seed cluster, there are shared endpoints for all shoots of a seed cluster.
+The main entry-point into the seed cluster is the loadbalancer of the Istio ingress-gateway service. Depending on the infrastructure provider, there can be one IP address per zone.
+
+The Istio ingress-gateway exposes the following TCP ports:
```

Comment:
Actually, these are the load balancer ports and not the Istio ingress-gateway ports. At least the pods use partially different ports.